dist/sizzle.js

```diff
@@ -1687,12 +1916,41 @@ tokenize = Sizzle.tokenize = function( selector, parseOnly ) {
 	// Return the length of the invalid excess
 	// if we're just parsing
 	// Otherwise, throw an error or return tokens
-	return parseOnly ?
-		soFar.length :
-		soFar ?
-			Sizzle.error( selector ) :
-			// Cache the tokens
-			tokenCache( selector, groups ).slice( 0 );
+	var validLen = soFar.length;
```

Comment:
I think `invalidLen` would better suit as a variable name, because `soFar` denotes a selector fragment that couldn't be parsed.

Reply:
++